website/docs/r/vapp_network.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_network"
+sidebar_current: "docs-vcd-resource-vapp-network"
+description: |-
+  Provides a vCloud Director vApp isolated Network. This can be used to create and delete internal networks for vApps to connect.
+---
+
+# vcd\_vapp\_network
+
+ Provides a vCloud Director vApp isolated Network. This can be used to create and delete internal networks for vApps to connect.
+ This network is not attached to external networks or routers.
+
+Supported in provider *v2.0+*
```

Comment:
Given that we are following semver, this new feature will bump the minor number, so we can point that in the comment above:
`Supported in provider *v2.1+*`

Reply:
Updated